schemas/adaptive-card.json

```diff
@@ -251,11 +421,38 @@
 						},
 						{
 							"$ref": "#/definitions/Input.ChoiceSet"
+						},
+						{
+							"$ref": "#/definitions/ActionSet"
+						},
+						{
+							"$ref": "#/definitions/RichTextBlock"
 						}
 					]
 				}
 			]
 		},
+		"ActionSet": {
+			"additionalProperties": true,
+			"type": "object",
+			"description": "Allow authors to place one or more 'action bars' anywhere within the card body",
```

Comment:
>Allow authors to place one or more 'action bars' anywhere within the card body [](start = 19, length = 78)

"Allows authors" is a bit strange. How about simply "Displays a set of actions.", which is consistent with "Displays an image" and "Displays text".